snuba/datasets/storages/generic_metrics.py

```diff
@@ -119,6 +119,7 @@ def produce_policy_creator() -> DeadLetterQueuePolicy:
         processor=GenericSetsMetricsProcessor(),
         default_topic=Topic.GENERIC_METRICS,
         dead_letter_queue_policy_creator=produce_policy_creator,
+        commit_log_topic=Topic.GENERIC_METRICS_SETS_COMMIT_LOG,
```

Comment:
You may want to move this line (and the distributions equivalent) into a separate PR. So we create the topics first and have them in topics.py but then start actually producing to them later.

Reply:
removing that now, and then I'll merge and then we can do the ops PR